packages/localization/src/service.ts

```diff
@@ -61,7 +62,12 @@ export class LocalizationService<T> extends (LocalizedStringsService as Localize
     languages: Language[];
     localeInfo: StringLocaleInfo;
 
-    constructor({ languages, localeInfo = defaultLocaleInfo }: LanguageServiceOptions<T>) {
+    constructor({
+        languages,
+        localeInfo = NativeModules.UIKitLocalization != null
+            ? NativeModules.UIKitLocalization.getConstants()
```

Comment:
How about creating an independent and exportable method for this?

Since we might also want to use it in Surf (once you remove the same native code from that side in this PR: https://github.com/tonlabs/TON-Surf/pull/2778/files)

Reply:
I actually think that it might be a good idea not to expose it, and use `localeInfo` directly :)

Reply:
But we use it pretty often in Surf (usually we call `TONString.getLocaleInfo()`)

To do so we have to setup our TONUtility with the proper localeInfo received from native

Reply:
Got it. OK, we'll switch on it.